src/main/java/org/schemaspy/output/dot/schemaspy/DotTableFormatter.java

```diff
@@ -206,39 +206,43 @@ private static Factory getFactory(Table table, boolean includeExcluded) {
         return factory;
     }
 
-    private static Set<Table> getTableImmediateRelatives(Table table, Factory factory, boolean includeImplied) {
-        Columns columns = factory.columns();
-        Set<TableColumn> relatedColumns = new HashSet<>();
+    private static Set<Table> immediateRelativesTables(Table table, Factory factory, boolean includeImplied) {
```

Comment:
Double plurals, since we are in the context of tables it seems is it a tables relatives tables? 2nd degree?
There are some strange things since tables relatives should be another table, and a columns relative should be a column I guess.

ImmediateRelative should be a table, and a table doesn't really have tables.